integration_tests/e2e/goal/createGoals.cy.ts

```diff
@@ -0,0 +1,57 @@
+import Page from '../../pages/page'
+import CreateGoalsPage from '../../pages/goal/CreateGoalsPage'
+import OverviewPage from '../../pages/overview/OverviewPage'
+
+context('Create goals', () => {
+  beforeEach(() => {
+    cy.task('reset')
+    cy.task('stubSignInAsUserWithEditAuthority')
+    cy.task('stubAuthUser')
+    cy.task('stubGetHeaderComponent')
+    cy.task('stubGetFooterComponent')
+    cy.task('stubPrisonerList')
+    cy.task('stubCiagInductionList')
+    cy.task('stubActionPlansList')
+    cy.task('getPrisonerById')
+    cy.task('stubGetInductionShortQuestionSet')
+    cy.task('getActionPlan')
+    cy.task('stubLearnerProfile')
+    cy.task('stubLearnerEducation')
+  })
+
+  it('should be able to navigate directly to Create Goal page', () => {
+    // Given
+    const prisonNumber = 'G6115VJ'
+    cy.signIn()
+
+    // When
+    cy.visit(`/plan/${prisonNumber}/goals/create`)
+
+    // Then
+    Page.verifyOnPage(CreateGoalsPage)
+  })
+
+  it('should not be able to submit Create Goal page given validation errors', () => {
+    // Given
+    const prisonNumber = 'G6115VJ'
+    cy.signIn()
+
+    cy.visit(`/plan/${prisonNumber}/view/overview`)
+    const overviewPage = Page.verifyOnPage(OverviewPage)
+    const createGoalPage = overviewPage.clickAddGoalButton()
+
+    createGoalPage //
```

Comment:
What's with the `//` here and one line 52? Is that a typo or is there a reason for it?